src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -12,17 +12,25 @@ public class NodeAutoCompleteSearchViewModel : SearchViewModel
     {
         internal PortViewModel PortViewModel { get; set; }
 
+        /// <summary>
+        /// Cache of default node suggestions, use it in case where
+        /// a. our algorithm does not return sufficient results
+        /// b. the results returned by our algorithm will not be useful for user
+        /// </summary>
+        internal IEnumerable<NodeSearchElementViewModel> DefaultResults { get; set; }
+
         internal NodeAutoCompleteSearchViewModel(DynamoViewModel dynamoViewModel) : base(dynamoViewModel)
         {
-            // Do nothing for now, but we may off load some time consuming operation here later
+            // Off load some time consuming operation here
+            InitializeDefaultAutoCompleteCandidates();
         }
 
         internal void InitializeDefaultAutoCompleteCandidates()
         {
             var candidates = new List<NodeSearchElementViewModel>();
             // TODO: These are hard copied all time top 7 nodes placed by customers
             // This should be only served as a temporary default case.
-            var queries = new List<string>(){ "Code Block", "Watch", "List Flatten", "List Create", "String", "Double", "Python" };
+            var queries = new List<string>(){ "Code Block", "Watch", "List.Flatten", "List.Create", "String", "Double", "Python" };
```

Comment:
@Amoursol do you agree with these default node results?

Reply:
@aparajit-pratap @Amoursol I also would like to discuss the expected behavior for CBN here because I remember the original behavior was CBN will be placed without connection, but seems some changes later broke that and now picking it does not do anything

Reply:
@QilongTang Thoughts as follows:

- While all of these are from the 'Most used nodes list' (Which is awesome to use as data-informed decision making!), some of them don't make sense.
- For example, the Watch Node is only used as output viewing, and the Codeblock node will just confuse new users and accessible swiftly on canvas with shortcut.
- My suggestions are to use the **Basic** set from the Node Library, in this order:
1. String
2. Number Slider
3. Integer Slider
4. Number
5. Boolean
- We need to make it clear that these results are a basic set, not real options or we may confuse users. 
- In the default state, can we style it differently? If so, please try italicize it.

Reply:
Updated the default options, these are currently shared UI style with right click search, we can consider restyle when we have own dedicated UI for Node autocomplete, what you think?